docs/proposals/synthetic-load-generator.md

```diff
@@ -0,0 +1,126 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: Introduce synthetic load generator tool<br/>
+authors: @HeavyWombat<br/>
+reviewers: TBD<br/>
+approvers: TBD<br/>
+creation-date: 2020-09-18<br/>
+last-updated: 2020-09-18<br/>
+status: provisional<br/>
+see-also: _n/a_<br/>
+replaces: _n/a_<br/>
+superseded-by: _n/a_<br/>
+
+---
+
+# Neat Enhancement Idea
+
+Introduce a tool to create synthetic load in a Kubernetes cluster using [shipwright-io/build] capabilities to help with performance tests or test plans.
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [docs](/docs/)
+
+## Open Questions [optional]
+
+This is where to call out areas of the design that require closure before deciding to implement the
+design. For instance:
+
+> 1. Use dynamic client or `code-gen` created `clientset`?
+> 1. What kind of default reports should be included, e.g. GraphJS, text, CSV?
+
+## Summary
+
+The performance of [shipwright-io/build] depends on the cluster capabilities and the actual application that is build into a container image. Using well-known synthetic applications for different cluster build strategies allows for reproducible execution times. In order to be able to detect performance regressions over time, it is useful to re-use the same test load all the time. Furthermore, the load generation should use the custom resrouce definition provided by `build` directly to avoid any additional overhead by a wrapper or convenienve layer.
```

Comment:
`the load generation should use the custom resrouce`-> `the load generation should use the custom resource`